Framework/include/QualityControl/AggregatorRunner.h

```diff
@@ -109,7 +109,7 @@ class AggregatorRunner : public framework::Task
   static framework::DataProcessorLabel getLabel() { return { "qc-aggregator" }; }
   static std::string createAggregatorRunnerIdString() { return "qc-aggregator"; };
   static std::string createAggregatorRunnerName();
-  static header::DataDescription createAggregatorRunnerDataDescription(const std::string& aggregatorName);
+  static header::DataDescription createAggregatorRunnerDataDescription(const std::string& aggregatorName, size_t hashSize = 4);
```

Comment:
```suggestion
  static header::DataDescription createAggregatorRunnerDataDescription(const std::string& aggregatorName);
```
This could be discussed, but I think it would be safer to assume always the same hash length for a given QC actor type. Otherwise, someone who wants to subscribe to any QC Data Processor would have to check the `InfrastructureGenerator` implementation (instead of just relying on information in a header) to use the correct hash length and would be exposed to incompatibilities if we change it.

I would propose that hashLength is a constant inside each of such `create*DataDescription` methods.

Reply:
sure, I will change it for all of the factory methods. I just saw there is written N for hash length in ticket, so I kinda propagated it everywhere...

Reply:
to be clear, the general function in `HashDataDescription` should still have the argument, the component-specific ones should use constants.